.ruby-version

```diff
@@ -1 +1 @@
-2.4
+2.4.2
```

Comment:
Please do not do that. We use RVM which works well without hardfixing the hotfix version.

Reply:
Ah yeah, didn't mean to check that in. I'm using rbenv locally and it didn't like that for some reason :roll_eyes:

Reply:
Yeah, rbenv unfortunately intentionally doesn't support this. You can workaround by symlinking a specific version to 2.4, I think there's some alias plugin which automates it a bit.